azext_iot/central/params.py

```diff
@@ -654,3 +661,261 @@ def load_central_arguments(self, _):
+            " [File Path Example:./path/to/attestation.json]"
+            " [Example of stringified JSON:[{<Attestation Data JSON>}]."
+            " The request body must contain GroupSymmetricKeyAttestation or GroupX509Attestation.",
+        )
+        context.argument(
+            "display_name",
+            options_list=["--display-name"],
+            help="Display name of the enrollment group."
+        )
+        context.argument(
+            "type",
+            options_list=["--type"],
+            help="Type of devices that connect through the group."
+        )
+        context.argument(
+            "enabled",
+            options_list=["--enabled"],
+            arg_type=get_three_state_flag(),
+            help="Whether the devices using the group are allowed to connect to IoT Central. True or False."
+        )
+        context.argument(
+            "certificate",
+            options_list=["--certificate", "--cert"],
+            help="The string representation of this x509 certificate.",
+        )
+        context.argument(
+            "verified",
+            options_list=["--verified", "--v"],
+            arg_type=get_three_state_flag(),
+            help="Whether the x509 certificate has been verified. True or False.",
+        )
+        context.argument(
+            "remove_x509",
+            options_list=["--remove-x509"],
+            arg_type=get_three_state_flag(),
+            help="Whether the x509 certificate should be removed from the group. True or False."
+        )
+        context.argument(
+            "etag",
+            options_list=["--etag"],
+            help="ETag used to prevent conflict in enrollment group updates."
+        )
+
+    with self.argument_context("iot central enrollment-group verify-x509") as context:
+        context.argument(
+            "certificate",
+            options_list=["--certificate", "--cert"],
+            help="The string representation of this x509 certificate.",
+        )
+
+    with self.argument_context("iot central scheduled-job") as context:
+        context.argument(
+            "job_id",
+            options_list=["--job-id", "--id"],
+            help="Unique identifier for the scheduled job.",
+        )
+
+    with self.argument_context("iot central scheduled-job create") as context:
+        context.argument(
+            "job_name",
+            options_list=["--job-name"],
+            help="Display name of the job.",
+        )
+        context.argument(
+            "group_id",
+            options_list=["--group-id", "-g"],
+            help="The ID of the device group on which to execute the job",
+        )
+        context.argument(
+            "schedule",
+            options_list=["--schedule"],
+            help="The schedule at which to execute the job. Provide path to JSON file or raw stringified JSON."
+            " [File Path Example:./path/to/file.json]"
+            " [Example of stringified JSON:[{<Schedule Data JSON>}].",
```

Comment:
these belong in help rather than help params

Reply:
I followed the old pattern we used before. But I can fix this.

Reply:
No I will still keep them here since these are explanations for params and help file only contains some command explanations.

Reply:
Fixed.